extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -163,30 +167,6 @@
 		},
 
 		/**
-		 * This runs after has signed in with facebook and is already registered with Wikia.
```

Comment:
We're doing all this in the backend now and just passing the exact URL to redirect to back with the AJAX response.